contracts/adapters/Configuration.sol

```diff
@@ -59,15 +60,30 @@ contract ConfigurationContract is
         bytes32 proposalId,
         bytes32[] calldata keys,
         uint256[] calldata values,
+        address[] calldata addresses,
         bytes calldata data
     ) external override onlyMember(dao) reentrancyGuard(dao) {
+        require(keys.length > 0, "missing keys");
         require(
-            keys.length == values.length,
-            "must be an equal number of config keys and values"
+            keys.length == values.length || keys.length == addresses.length,
```

Comment:
The idea was to have at least one of them matching because it is possible to pass only one of them to be updated.  Hence if we force both to be declared we would be adding both types of configs into the registry, but we don't really want that we want to add only a specific type of config. The way it was implemented was to prevent it from explicitly passing a type. Considering the case you want to update only the `address` configs , you won't need to inform the `values` array.